app/models/condition.rb

```diff
@@ -87,7 +87,10 @@ def self.evaluate(cond, rec, inputs={}, attr=:expression)
   def self.do_eval(expr)
     # Remove this [DEPRECATION] in the next version
     if expr =~ /^__start_ruby__\s?__start_context__\s?(.*)\s?__type__\s?(.*)\s?__end_context__\s?__start_script__\s?(.*)\s?__end_script__\s?__end_ruby__$/im
-      raise "Ruby scripts in expressions are no longer supported. Please use the regular expression feature of conditions instead."
+      context, col_type, script = [$1, $2, $3]
+      context = MiqExpression.quote(context, col_type)
+      result = SafeNamespace.eval_script(script, context)
```

Comment:
@matthewd We are going to implement a better solution. Just can't do it within the scope of the BZ. In order to implement a better solution we'll need to redesign some of the very crufty internals of the expression engine and that's a longer term exercise.